system/HTTP/IncomingRequest.php

```diff
@@ -707,7 +707,7 @@ public function getRawInput()
      * @param int|null          $filter Filter Constant
      * @param array|int|null    $flags  Option
      *
-     * @return mixed
+     * @return array|bool|int|object|null
```

Comment:
Why? Basically the value is a string, and an array if `$index` is an array.

It can get `$filter`, so `FILTER_CALLBACK` can return any type value.
https://www.php.net/manual/en/filter.filters.misc.php

It seems the return type is `array|bool|float|int|object|string|null`.

Reply:
Any `get*()` method return type is the same?

Reply:
Any `get*()` method return type is the same.

```php
dd($this->request->getGetPost('test', FILTER_VALIDATE_FLOAT));
```
Navigate to `http://localhost:8080/?test=1.11` and I got:
```
$this->request->getGetPost(...) double 1.11
```

The return type of `fetchGlobal()` also should be `array|bool|float|int|object|string|null`.